modules/poop/index.js

```diff
@@ -127,6 +135,11 @@ module.exports = class Poop extends BaseStorageModule {
     return ['poop-board', 'poopboard', ':poop:', 'poops'];
   }
 
+  isBlackListed(channel) {
+    let channels = ['announcements', 'incident-discussion', 'product-chat']
```

Comment:
This can be const and lets move it to the config since most workspaces don't care about channels.  Here is an example: https://github.com/konecnyna/slack-cat/blob/master/modules/pagerduty/index.js#L76

Reply:
configed.